test/spec/algoliasearch.helper/numericFilters.js

```diff
@@ -73,113 +72,80 @@ test('Numeric filters: numeric filters from setters', function(t) {
-  t.end();
+  expect(helper.state.numericRefinements.attribute).toBe(undefined);
 });
 
 test(
   'Should remove all the numeric values for a single operator if remove is called with two arguments',
-  function(t) {
+  function() {
     var helper = algoliasearchHelper(fakeClient, null, null);
 
     helper.addNumericRefinement('attribute', '>', 0);
     helper.addNumericRefinement('attribute', '>', 4);
     helper.addNumericRefinement('attribute', '<', 4);
-    t.deepEqual(
-      helper.state.numericRefinements.attribute,
-      {'>': [0, 4], '<': [4]},
-      'should be set to {">": [0, 4], "<":[4]} initially');
+    expect(helper.state.numericRefinements.attribute).toEqual({'>': [0, 4], '<': [4]});
     helper.removeNumericRefinement('attribute', '>');
-    t.equal(helper.state.numericRefinements.attribute['>'], undefined, 'should set to undefined');
-    t.deepEqual(
-      helper.state.numericRefinements.attribute['<'],
-      [4],
-      'the value of the other should be the same');
-
-    t.deepEqual(
-      helper.getRefinements('attribute'),
-      [{type: 'numeric', operator: '<', value: [4]}],
-      'should have the same result with getRefinements');
-
-    t.end();
+    expect(helper.state.numericRefinements.attribute['>']).toBe(undefined);
+    expect(helper.state.numericRefinements.attribute['<']).toEqual([4]);
+
+    expect(helper.getRefinements('attribute')).toEqual([{type: 'numeric', operator: '<', value: [4]}]);
   }
 );
 
 test(
   'Should remove all the numeric values for an attribute if remove is called with one argument',
-  function(t) {
+  function() {
     var helper = algoliasearchHelper(fakeClient, null, null);
 
     helper.addNumericRefinement('attribute', '>', 0);
     helper.addNumericRefinement('attribute', '>', 4);
     helper.addNumericRefinement('attribute', '<', 4);
-    t.deepEqual(
-      helper.state.numericRefinements.attribute,
-      {'>': [0, 4], '<': [4]},
-      'should be set to {">": [0, 4], "<":[4]} initially');
+    expect(helper.state.numericRefinements.attribute).toEqual({'>': [0, 4], '<': [4]});
     helper.removeNumericRefinement('attribute');
-    t.equal(helper.state.numericRefinements.attribute, undefined, 'should set to undefined');
+    expect(helper.state.numericRefinements.attribute).toBe(undefined);
 
-    t.deepEqual(
-      helper.getRefinements('attribute'),
-      [],
-      'should have the same result with getRefinements');
-
-    t.end();
+    expect(helper.getRefinements('attribute')).toEqual([]);
   }
 );
 
-test('Should be able to get if an attribute has numeric filter with hasRefinements', function(t) {
+test('Should be able to get if an attribute has numeric filter with hasRefinements', function() {
   var helper = algoliasearchHelper(fakeClient, null, null);
 
-  t.notOk(helper.hasRefinements('attribute'), 'not refined initially');
+  expect(helper.hasRefinements('attribute')).toBeFalsy();
```

Comment:
Can be more strict.

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453